addons/mail/static/src/components/autocomplete_input/autocomplete_input.js

```diff
@@ -15,12 +15,6 @@ export class AutocompleteInput extends Component {
         useComponentToModel({ fieldName: 'component', modelName: 'AutocompleteInputView' });
         onMounted(() => this._mounted());
         onWillUnmount(() => this._willUnmount());
-        useEffect(() => {
-            if (this.props.inputRef) {
-                this.props.inputRef.el = this.root.el;
-                return () => this.props.inputRef.el = null;
-            }
-        })
```

Comment:
Can you explain what the purpose was and why it is no longer necessary?

Reply:
There was a trick to pass the ref, in order to focus the autocomplete input.
For some reasons it relies on `useEffect` that is a bit like `onMounted`/`onWillDestroy` at the same time.

Now that inputRef is moved to models thanks to component hooks, and the component hooks handle setting and clearing the field in models according to component lifecycle, it's no longer necessary to rely on this props to manually set and clear the ref.